pkg/epp/scheduling/types/types.go

```diff
@@ -79,10 +87,15 @@ func NewContext(ctx context.Context, req *LLMRequest, pods []*PodMetrics) *Conte
 	}
 }
 
-func ToSchedulerPodMetrics(pods []backendmetrics.PodMetrics) []*PodMetrics {
-	pm := make([]*PodMetrics, 0, len(pods))
+func ToSchedulerPodMetrics(pods []backendmetrics.PodMetrics) []Pod {
+	pm := make([]Pod, 0, len(pods))
```

Comment:
Since the return type is changed, consider renaming the var from `pm` to `p` or something similar to the return type.